src/cfg/cfg-traversal.h

```diff
@@ -278,9 +279,13 @@ struct CFGWalker : public PostWalker<SubType, VisitorType> {
     // catch $e3
     //   ...
     // end
-    assert(self->unwindExprStack.size() == self->throwingInstsStack.size());
+    assert(self->tryStack.size() == self->throwingInstsStack.size());
     for (int i = self->throwingInstsStack.size() - 1; i >= 0;) {
-      auto* tryy = self->unwindExprStack[i]->template cast<Try>();
+      // Exception thrown. Note outselves so that we will create a link to each
+      // catch within the try when we get there.
+      self->throwingInstsStack[i].push_back(self->currBasicBlock);
```

Comment:
This code movement is not obviously NFC to me. Before it was not reached if we got to the break or continue on new lines 293 and 307?

Reply:
Hmm, you're right. Will revert this.